src/SquareEdit/Contents.ts

```diff
@@ -81,6 +81,7 @@ export const editControlsPanel: string =
     '<div class=\'field is-grouped buttons\'>' +
     '<p class=\'control\'>' +
     '<a id=\'moreEdit\' class=\'panel-block is-invisible\'>' +
+    '<a id=\'extraEdit\' class=\'panel-block is-invisible\'>' +
```

Comment:
These should have more descriptive names or at least an explanation distinguishing what `moreEdit` and `extraEdit` mean until we rewrite this anyway.